src/components/Header/Header.js

```diff
@@ -1,38 +1,37 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './Header.scss';
-import { Link } from 'react-router-dom';
+import { Link, NavLink } from 'react-router-dom';
 import logo from '../../assets/lienflash-logo.png'
 
 function Header(props) {
   const { currentPath } = props;
 
-  const handleClick = (event) => {
-    if (event.target.nextSibling) {
-      event.target.nextSibling.classList.remove('selected')
-    } else if (event.target.previousElementSibling) {
-      event.target.previousElementSibling.classList.remove('selected')
-    }
-
-    event.target.classList.add('selected')
-
-    // need to also switch view to those cards; set button to be a link
-  }
-
   return (
     <header>
       <Link to={'/'}>
         <img src={logo} className='logo' alt='lienflash logo'/>
       </Link>
         {currentPath !== undefined && currentPath === 'filed'&&
-            <div onClick={handleClick}>
-              <button className='in-process-btn'>In Process</button>
-              <button className='release-lien-eligible-btn'>Release Eligible</button>
+            <div>
+              <NavLink to={'/filedjobs/processing'} className='nav-button' activeClassName="selected">
+                In Process
+              </NavLink>
+              <NavLink to={'/filedjobs/release-eligible'} className='nav-button' activeClassName="selected">
+                Release Eligible
+              </NavLink>
             </div>
         }
         {currentPath !== undefined && currentPath === 'eligible'&&
-            <div onClick={handleClick}>
-              <button className='noi-eligible-btn'>NOI Eligible</button>
-              <button className='lien-eligible-btn'>Lien Eligible</button>
+            <div>
+              <NavLink to={'/eligiblejobs/grace-period'} className='nav-button' activeClassName="selected">
+                Grace Period
+              </NavLink>
+              <NavLink to={"/eligiblejobs/noi-eligible"} className='nav-button' activeClassName="selected">
+                NOI Eligible
+              </NavLink>
+              <NavLink to={"/eligiblejobs/lien-eligible"} className='nav-button' activeClassName="selected">
+                Lien Eligible
+              </NavLink>
             </div>
```

Comment:
Love the NavLink solution for showing the active 'button'. Super clean implementation!